### DISCUSSION TITLE
Quasar Vue/Cli version does not work

### DISCUSSION THREAD

**Discussion by murugappanrm** [quasarframework/quasar#12943] [CLI - SSR mode] [answered]
I using vue/cli version 5.0.1 and quasar/cli version 1.3.2. I install quasar using the vue/cli option. The install was successful but when i run "npm run serve" the page created is messed-up: 

![Screenshot1](https://user-images.githubusercontent.com/63133893/160226933-be5c9607-d3a2-4b57-821f-715f7bacc01d.png)

Please help.

**Comment by smolinari** [comment]
I can confirm the issue. Creating an issue from this discussion. 

Scott

**Reply by smolinari** [reply]
@murugappanrm - It seems the issue is known actually. See here:  https://github.com/quasarframework/vue-cli-plugin-quasar/issues/52

Scott

**Comment by yusufkandemir** [comment]
Vue CLI Plugin Quasar doesn't support Vue CLI 5 **yet**. Follow this issue for details/progress: https://github.com/quasarframework/vue-cli-plugin-quasar/issues/52

Also keep in mind that Vue CLI is now in maintenance mode: https://github.com/vuejs/vue-cli/tree/v5.0.4#status

**Reply by smolinari** [reply]
Hehe... a smidge faster than me. :P

Scott

**Comment by rstoenescu** [comment, marked as answer]
Released vue-cli-plugin-quasar v5.0.1 which has support for Vue CLI v5. https://github.com/quasarframework/vue-cli-plugin-quasar/releases/tag/v5.0.1
